gulpfile.js

```diff
@@ -88,7 +88,8 @@ gulp.task('types', function(done) {
   var files = ['browser', 'element', 'locators', 'expectedConditions',
     'config', 'plugins', 'ptor'];
   var outputFile = path.resolve(folder, 'index.d.ts');
-  var contents = '';
+  var contents = '/// <reference path="../typings/index.d.ts" />\n';
```

Comment:
As discussed, triple slash reference path brings in the ambient typings. We are doing this specifically for selenium-webdriver typings.